docs/openapi.yml

```diff
@@ -0,0 +1,37 @@
+openapi: "3.0.2"
```

Comment:
Not done with this, but I figured it didn't need to hold up the rest of the review.